README.md

```diff
@@ -49,15 +60,107 @@ environment variable `DSS_GS_BUCKET_TEST_FIXTURES` to the name of that bucket.
 
 **Azure**: Set the environment variables `AZURE_STORAGE_ACCOUNT_NAME` and `AZURE_STORAGE_ACCOUNT_KEY`.
 
-#### Running the prototype
-Run `./dss-api` in this directory.
+#### Running the DSS API locally
+Run `./dss-api` in the top-level `data-store` directory.
+
+#### Populate test data
+
+To run the tests, test fixture data must be setup using the following command.
+**This command will completely empty the given buckets** before populating them with test fixture data, please 
+ensure the correct bucket names are provided.
+
+    python tests/fixtures/populate.py --s3-bucket $DSS_S3_BUCKET_TEST_FIXTURES --gs-bucket $DSS_GS_BUCKET_TEST_FIXTURES
+
 
 #### Running tests
-Run `make test` in this directory.
 
-Some tests require the Elasticsearch service to be running on the local system.
+Some tests require the Elasticsearch service to be running on the local system:
+
 Run: `elasticsearch`
 
+Then to perform the data store tests:
+
+Run `make test` in the top-level `data-store` directory.
+
+#### Deployment
+
+Assuming the tests have passed above, the next step is to manually deploy.  See the section below for information on CI/CD with Travis if continuous deployment is your goal.
+
+You will need to ensure you are ready for deployment refer to `.travis.yml` to ensure you have all needed requirements.  Currently, the Travis deployment uses Ubuntu Trusty (14.04) with the following packages apt-get installed:
+
+    sudo apt-get install jq moreutils gettext
```

Comment:
In addition to moving these up to before the test instructions, I recommend removing these instructions from this file and replacing them with an instruction to call `make --dry-run`. This way you don't duplicate the dependency checking functionality that is now built into the makefile. (Lists of dependencies in readmes inevitably get out of date)